app/client/__tests__/components/identity/MarketingCheckbox.tsx

```diff
@@ -0,0 +1,53 @@
+import { cleanup, render, fireEvent, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import React from "react";
+import { MarketingCheckbox } from "../../../components/identity/MarketingCheckbox";
+
+afterEach(cleanup);
+
+describe.only("MarketingCheckbox component", () => {
```

Comment:
`.only` is still used here. Maybe should add some linting for that? (in a follow up PR).

Reply:
oops. fixed. Yes, will look at adding now in a new pr